### ISSUE TITLE
Errors in Reproducing MIOU values

### ISSUE THREAD

**Issue by aliman80** [dbash/zerowaste#5]
Hi, Thank you for your wonderful work. This is the kind of work i was looking .I am trying to reproduce these results  using deeplab but consistently my results are less in mIOU while for pACC they are ok. Similarly when i train model using RCNN my AP values are consistently less like i am g etting 17 instead of 22 and so on. Can you guide me if there is any hyperparamter which i needs to set or am i missing some thing. 
Thanking you in advance.

**Comment by dbash** [comment]
Hi Muhammad, thank you for your interest in our project. 

First, have you tried using a pretrained checkpoint we provide for DeepLab with the corresponding configs, does this result in the same accuracy as reported in the paper? If not, there might be an issue with the config / loading the checkpoint. 

If you are training from scratch, it is important to keep the same hyperparameters, especially the batch size.

**Comment by dbash** [comment]
Hi Muhammad, thank you for your interest in our project.

First, have you tried using a pretrained checkpoint we provide for DeepLab with the corresponding configs, does this result in the same accuracy as reported in the paper? If not, there might be an issue with the config / loading the checkpoint.

If you are training from scratch, it is important to keep the same hyperparameters, especially the batch size.

**Comment by aliman80** [comment]
Thank you for your response. Yes i treid your pretrained checkpoint it shows the same result as reported in the paper. But training from scratch is giving me loww accuracy i am forced to keep the batch size 8 as i have single GPU.

Thanks

**Comment by dbash** [comment]
Keeping the batch size large is important for the overall segmentation quality. I suggest using the same batch size (or larger) as in our experiments to reproduce the results. If your computational resources are limited you can use some techniques such as gradient accumulation.
